docs/how-tos/upload-module.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "How to upload a module"
+linkTitle: "Upload a module"
+type: "docs"
+weight: 26
+images: ["/registry/module-puzzle-piece.svg"]
+tags: ["modular resources", "components", "services", "registry"]
+description: "Make your module available to others by uploading it to the modular resource registry."
+aliases:
+  - /registry/upload/
+  - /extend/modular-resources/upload/
+  - /modular-resources/upload/
+---
+
+After [writing a module](/how-tos/create-module/), you can upload your module to the Viam registry to:
```

Comment:
Missing prerequisites section and missing in this page card

Reply:
Okay will add prereqs but I temporarily added an in this page card and then took it away again because it's only one item, with nothing to link to, so it just looks weird and distracts from the intro paragraph.

Reply:
Adding it anyway--lmk if you don't like it